extension/options/public/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
I think it makes sense to just move this to the root of the `extension` directory (unless we want to create a `public` directory there?).

Reply:
It's now in `extension/src/options/index.html`. Since it's only used for options, I don't think it feels right to have it at the top level.

Reply:
If it's going to live in `src`, can we use [CopyWebpackPlugin](https://webpack.js.org/plugins/copy-webpack-plugin) to copy it into `dist`?

Reply:
That could work, but I feel like it's only worth it if we're just publishing `dist`.

Reply:
To be clearer, I don't think we should be shipping anything in `src`. If things don't need to be run through webpack, I would advocate for either using the webpack plugin above or putting them in a directory like `extension/{public,assets}`.

Reply:
I see what you mean. Would you prefer to do this before merging/publishing the options page?

Reply:
I’m happy to implement it after.